myokit/tests/test_sbml_export.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+#
+# Tests Myokit's SBML support.
+#
+# This file is part of Myokit.
+# See http://myokit.org for copyright, sharing, and licensing details.
+#
+import unittest
+
+import myokit
+import myokit.formats
+
+import myokit.formats.sbml
+from myokit.formats.sbml._api import Model
+from myokit.formats.sbml._writer import write_string
+import myokit.formats.sbml as sbml
+
+
+class TestSBMLExport(unittest.TestCase):
+    """
+    Unit tests for the SBML export functionality.
+    """
+
+    def test_empty_model(self):
+        # Test exporting an empty model
+        model = Model()
+        sbml_str = write_string(model).decode("utf8")
+        self.assertIn("<sbml", sbml_str)
+        self.assertIn("</sbml>", sbml_str)
+        self.assertIn('<model id="unnamed_model"/>', sbml_str)
+
+    def test_time_unit(self):
+        # Test setting the time unit
+        model = Model()
+        model.set_time_units(myokit.units.second)
+        sbml_str = write_string(model).decode("utf8")
+        self.assertIn('timeUnits="second"', sbml_str)
+
+        model = Model()
+        model.set_time_units(1e3 * myokit.units.second)
```

Comment:
Who needs hours when we've got kiloseconds! Gonna see if I can get a watch with these